### ISSUE TITLE
lyricwiki plugin: GObject Introspection: Namespace WebKit not available

### ISSUE THREAD

**Issue by goetzc** [quodlibet/quodlibet#1663]
Maybe related to issue #1452, under Quod Libet 3.4.1, I see this error in the Plugins Errors window. I have webkitgtk 2.4.9-1 (from Arch Linux) installed. 

Not sure why this happens, considering commit 6c878227 which is included in QL 3.4.1.

In a Python 2 interpreter, this works:
```
>>> import gi
>>> gi.require_version("WebKit", "3.0")
````

**Comment by lazka** [comment]
What is the exact error shown in the plugin window?

**Comment by goetzc** [comment]
The error was the same as the issue title. 
```"GObject Introspection: Namespace WebKit not available"```
I installed webkitgtk after I first saw the error, and it still showed the message. I'm sure QL was closed and fresh started again after the error...

Now it works, but the error displayed in QL didn't specify a required version, which I though was odd.

I also have the following Plugin errors, without a required version:
![ql_plugin_error](https://cloud.githubusercontent.com/assets/2220440/9156760/c7c97cc2-3eaa-11e5-98f9-1c20393a8e67.png)

```
crossfeed
Couldn't find GStreamer element 'bs2b'.

kakasi
Couldn't find the 'Kanji Kana Simple Inverter' (kakasi).

zeitgeist_client
Couldn't find module 'zeitgeist'. Perhaps you need to install the package?
```
